### ISSUE TITLE
Cannot pass aria-label into Checkbox component

### ISSUE THREAD

**Issue by daweffe** [OfficeDev/office-ui-fabric-react#2410]
### Bug Report

- __Package version(s)__: 4.21.3
- __Browser and OS versions__: all

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Low

Products/sites affected: Planner

#### Describe the issue:

I am trying to get the screen reader to read more information than we show in the label text by passign in an aria label, but I cannot figure out how.  The inputProps object will not accept ariaLabel as a parameter.

**Comment by mdahamiwal** [comment]
Thanks for catching this. However, in your case you should use `ariaDescribedBy`, using ariaLabel will override the `label` props which might not be desired behavior.

**Comment by daweffe** [comment]
ariaDescribedBy and ariaLabeledBy also do not work on the checkbox element.

I do want to override the label text for screen readers.  The label text is “Show on card” which we show in several locations.  The aria label text adds “Show X on card,” where X is replaced with checklist, description, or an attachment filename.  I only want the screen reader to read the latter string, not both.
